<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="MSI" failures="1" tests="2" name="Test" time="14.906" errors="0" timestamp="2025-02-16T10:59:04 IST">
  <testcase classname="testCases.tc001_AccountRegistrationPage" name="verify_account_regirtration" time="3.092">
    <failure type="java.lang.AssertionError" message="expected [Congratulations! Your new account has been successfully created!] but found [If you already have an account with us, please login at the login page.]">
      <![CDATA[java.lang.AssertionError: expected [Congratulations! Your new account has been successfully created!] but found [If you already have an account with us, please login at the login page.]
at testCases.tc001_AccountRegistrationPage.verify_account_regirtration(tc001_AccountRegistrationPage.java:46)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- verify_account_regirtration -->
  <testcase classname="testCases.LoginTest" name="verifyLogin" time="2.049"/>
</testsuite> <!-- Test -->
